h2/src/main/org/h2/mvstore/DataUtils.java

```diff
@@ -918,10 +919,15 @@ public static String formatMessage(int errorCode, String message,
                 arguments[i] = s;
             }
         }
-        return MessageFormat.format(message, arguments) +
-                " [" + Constants.VERSION_MAJOR + "." +
-                Constants.VERSION_MINOR + "." + Constants.BUILD_ID +
-                "/" + errorCode + "]";
+
+        try {
+            return MessageFormat.format(message, arguments) +
```

Comment:
It doesn't look like a proper place for a fix. `MVStore.store()` passes an invalid argument here and it should be fixed instead.

Reply:
I just put PR #1872 to fix that

Reply:
In that case I should close this PR.